### ISSUE TITLE
Better support for Pleroma and forks (Akkoma, etc)

### ISSUE THREAD

**Issue by draga79** [elk-zone/elk#672]
Elk is nice and promising. Many users don't like the original Mastodon UI or the frontends available for Pleroma and forks. 
It would be nice if elk could support Pleroma and Akkoma's extended functionalities and push people to be able to choose between the heavy Mastodon and lighter alternatives.
This could make elk the best web client not only for Mastodon, but also for the other fediverse microblogging platforms.

**Comment by gerwitz** [comment]
It doesn't seem to work at all with my Akkoma instance. I assume by this issue that this is an "only me" problem?

**Comment by draga79** [comment]
> 

Nope, it was working, but the last updates broke the Akkoma/Pleroma/GoToSocial/etc. compatibility :(

**Comment by gerwitz** [comment]
It's letting me login to Akkoma now, on iOS Safari 👍

**Comment by yheuhtozr** [comment]
I'm interested in investigating (and hopefully implementing) it, but is there any list of missing features for Pleroma et al. compatibility?

**Comment by emma-sg** [comment]
Still having issues with my Akkoma instance – at the moment the issue seems to be with oauth, because I'm getting a "Missing authentication code" error message when trying to log in.

**Comment by rjp** [comment]
Can confirm that the "Missing authentication code" is still a problem with a newly installed Akkoma.
